frontend/src/component/admin/network/NetworkOverview/NetworkOverview.tsx

```diff
@@ -22,24 +22,33 @@ export const NetworkOverview = () => {
     const { metrics } = useInstanceMetrics();
 
     const apps: INetworkApp[] = [];
-    if (Boolean(metrics)) {
```

Comment:
Why the coercion to Boolean? isn't it enough to say `if (metrics) {}`. I'm genuinely curious, not trying to enforce one or the other.

Reply:
I don't remember the rationale behind it, but I'm pretty sure you're correct that it's not doing anything and should be cleaned up.